README.md

```diff
@@ -161,6 +161,12 @@ expect(context.component()).toBeInstanceOf(MyComponent);
 ### `FindWrapper#contains(vdom)`
 Searches for any children matching the vdom or text passed.
 
+### `FindWrapper#childAt(index)`
+Returns `FindWrapper` with child at given index.
+
+### `FindWrapper#children()`
+Returns `FindWrapper` with children of current wrapper.
```

Comment:
Perhaps a small usage example if you can think of one?

Reply:
Done